### PR TITLE
feat(docs): final touch + fix url

### PR DESCRIPTION
**Pull request by torgeadelin**
# Description
- final touches
- home page redesign
- docs url fixes

### CODE REVIEW

File: packages/@momentum-design/docs/src/components/RightSidebar/MoreMenu.astro [hunk 1]
```diff
@@ -17,7 +16,7 @@ const lang = getLanguageFromURL(currentPage);
 
 {showMoreSection && <h2 class="heading">More</h2>}
 <ul>
-  {
+  <!-- {
```
Comment (chrisadubois) [inline packages/@momentum-design/docs/src/components/RightSidebar/MoreMenu.astro:19]: delete code?

File: packages/@momentum-design/docs/src/pages/en/assets/index.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Assets
```
Comment (chrisadubois) [inline packages/@momentum-design/docs/src/pages/en/assets/index.mdx:2]: frontmatter !

**Review by chrisadubois — APPROVED** [review]

File: packages/@momentum-design/docs/src/pages/en/assets/index.mdx (hunk 2 above)
Comment (torgeadelin) [reply packages/@momentum-design/docs/src/pages/en/assets/index.mdx:2]: that is required by Astro, for the layout hehe

File: packages/@momentum-design/docs/src/components/RightSidebar/MoreMenu.astro (hunk 1 above)
Comment (torgeadelin) [reply packages/@momentum-design/docs/src/components/RightSidebar/MoreMenu.astro]: deleted

**Review by chrisadubois — APPROVED** [review]

File: packages/@momentum-design/docs/src/pages/en/assets/index.mdx (hunk 2 above)
Comment (chrisadubois) [reply packages/@momentum-design/docs/src/pages/en/assets/index.mdx:2]: i know! its great !

**Pull request merged**